afqbrowser/site/client/js/table.js

```diff
@@ -64,14 +64,22 @@ afqb.table.buildTable = function (error, useless, data) {
             return !isNaN(+e);
         }
 
+        function isInt(e) {
+            return Number.isInteger(+e);
+        }
+
         function identity (arg) {
             return arg;
         }
 
         if (column.every(isBinary)) {
             afqb.table.subFormats[key] = d3.format("0b");
         } else if (column.every(isNum)) {
-            afqb.table.subFormats[key] = d3.format(".7g");
+            if (column.every(isInt)) {
+                afqb.table.subFormats[key] = d3.format("d");
+            } else {
+                afqb.table.subFormats[key] = d3.format(".5");
```

Comment:
Won't this show 5 digits after the decimal? I thought @jyeatman preferred 2

Reply:
Argh, I thought I tested this well enough, but evidently not. I was using d3.v4 format specifiers instead of d3.v3. I just pushed a new commit to fix it. The intended behavior is to use 4 decimal places unless the data uses less, in which case it trims the trailing zeros. I thought this was in line with @jyeatman's [comment here](https://github.com/yeatmanlab/AFQ-Browser/issues/247#issuecomment-352935967) since it uses the number of decimals defined in the data. The fallback case of four decimal places is more permissive than his suggested two decimal places but I read his comment as "use two decimal places if you can't figure out how to drop the trailing zeros". Since I found a way to drop the trailing zeros, then I thought four would be better. @jyeatman, did I interpret your comment correctly?